app/src/main/java/com/alorma/github/ui/fragment/detail/repo/SourceListFragment.java

```diff
@@ -206,6 +219,59 @@ public void onContentMenuAction(Content content, MenuItem menuItem) {
         Intent intent = ContentCommitsActivity.createLauncherIntent(getActivity(), repoInfo, content.path, content.name);
         startActivity(intent);
         break;
+      case R.id.action_content_download:
+        if (file.equals(content.type)) {
```

Comment:
Plz extract to a method, so it's easy to read.

Soon i will move all of this methods to a Presenter class